docs/decisions/0006-middleware-support-for-http-clients.rst

```diff
@@ -0,0 +1,45 @@
+Middleware Support for HTTP clients
+===================================
+
+Status
+------
+
+Proposed
+
+Context
+-------
+
+We currently expose HTTP clients(axios instances) via ``getAuthenticatedHttpClient`` and ``getHttpClient`` used in
+our MFEs to make API requests. Since Open edX uses Django for its backend services, the MFEs have to handle snake-cased <-> camelCase
```

Comment:
This is a nit, but the ADR is about middleware support in general - may be worth framing this snake_case <-> camelCase thing as an example of a possible useful middleware.  As the second sentence in the context, it feels a bit out of nowhere.